packages/ERTP/core/config/inviteConfig.js

```diff
@@ -7,10 +7,7 @@ import { insist } from '../../util/insist';
 import { mustBeComparable } from '../../util/sameStructure';
 
 const insistOptDescription = optDescription => {
-  insist(
-    !!optDescription,
-  )`Uni optDescription must be either null or truthy: ${optDescription}`;
-  insist(!!optDescription)`Uni description must be truthy ${optDescription}`;
+  insist(!!optDescription)`optDescription must be truthy ${optDescription}`;
```

Comment:
Oh sorry, I didn't notice that this was waiting on me!

This code is confusing --- both the before and after. The `extentOps` in question is https://github.com/Agoric/agoric-sdk/blob/master/packages/ERTP/core/config/extentOps/uniExtentOps.js, for which null is a valid extent. But the uniExtentOps' `insistKind` method first checks for null, and only calls the `customInsistKind` if it is non-null.

The `insistOptDescription` is provided as this `customInsistKind` argument, so it will never be called with null. Thus, your change is correct. But please also remove the "opt" from `insistOptDescription` and `optDescription`, as it implies a null tolerance that these don't and shouldn't have.

However, we have a worse problem which I may have introduced. Attn @katelynsills . The inviteConfig describes the extentOps as

```js
    extentOpsName: 'inviteExtentOps',
    extentOpsArgs: [insistOptDescription],
```

which is therefore how the inviteAssay describes it. But this doesn't remote correctly. The problem is that the `extentOpsArgs` must be transmitted by copy, but `insistOptDescription` is a function. The real problem is `makeUniExtentOps` function, which I designed to take a function argument.

I'll file a separate bug on that. Even though they both touch this code, you should proceed anyway (with the renaming), as your change only makes things more correct and doesn't affect the nature of this bug.